Examples/Package.swift

```diff
@@ -51,10 +51,6 @@ let package = Package(
   ]
 )
 
-// The following targets are added only if the OS is macOS. This is because Swift macros are currently
-// not available on other platforms. As a result, we're guarding these targets with `#if os(macOS)`
-// to ensure that they are only included in the package when building on a macOS system.
-#if os(macOS)
 package.targets.append(
```

Comment:
Apologies for the delayed comment, but upon reflection, I noticed that the current way of adding targets seems unnecessary given the changes. How about defining the targets directly within the `Package` initializer, like the others? Here's a code snippet for your convenience:
```swift
  targets: [
    .executableTarget(
      name: "AddOneToIntegerLiterals",
      dependencies: [
        .product(name: "SwiftParser", package: "swift-syntax"),
        .product(name: "SwiftSyntax", package: "swift-syntax"),
      ]
    ),
    .executableTarget(
      name: "CodeGenerationUsingSwiftSyntaxBuilder",
      dependencies: [
        .product(name: "SwiftSyntaxBuilder", package: "swift-syntax")
      ]
    ),
    .macro(
      name: "MacroExamplesImplementation",
      dependencies: [
        .product(name: "SwiftCompilerPlugin", package: "swift-syntax"),
        .product(name: "SwiftSyntax", package: "swift-syntax"),
        .product(name: "SwiftSyntaxMacros", package: "swift-syntax"),
        .product(name: "SwiftDiagnostics", package: "swift-syntax"),
      ],
      path: "Sources/MacroExamples/Implementation"
    ),
    .target(
      name: "MacroExamplesInterface",
      dependencies: [
        "MacroExamplesImplementation"
      ],
      path: "Sources/MacroExamples/Interface"
    ),
    .executableTarget(
      name: "MacroExamplesPlayground",
      dependencies: [
        "MacroExamplesInterface"
      ],
      path: "Sources/MacroExamples/Playground"
    ),
    .testTarget(
      name: "MacroExamplesImplementationTests",
      dependencies: [
        "MacroExamplesImplementation",
        .product(name: "SwiftSyntaxMacros", package: "swift-syntax"),
        .product(name: "SwiftSyntaxMacrosTestSupport", package: "swift-syntax"),
      ],
      path: "Tests/MacroExamples/Implementation"
    ),
  ]
```
Feel free to copy and paste this into your implementation if you get the chance.

Reply:
I see! Thanks for the respond!